Util/channel/src/main/java/io/deephaven/util/channel/SeekableChannelContextNull.java

```diff
@@ -13,7 +13,7 @@ enum SeekableChannelContextNull implements SeekableChannelContext {
 
     @Override
     @Nullable
-    public SafeCloseable apply(final Supplier<SafeCloseable> resourceFactory) {
+    public SafeCloseable apply(final String key, final Supplier<SafeCloseable> resourceFactory) {
```

Comment:
This would be nicer as:
```
public <T extends SafeCloseable> T apply(final String key, final Supplier<T> resourceFactory)
```

I think the reason it isn't was to make it nicer when passed to the `CompressorAdapter`. I'm not sure if that consideration will stand after this PR.

Reply:
I think it would be nice to not give the decompressor adapter access to the map. Could change that interface back to `Function<Supplier<SafeCloseable>, SafeCloseable>` and supply the key from the caller, e.g.
```
scf -> holder.get().apply(DECODER_CACHE_KEY, scf)
```